flyingsnake/__main__.py

```diff
@@ -116,64 +116,58 @@ def flyingsnake(input_file: str,
         c.echo("Drawing walls...")
         walls = Image.new("RGBA", (world.size.x, world.size.y))
         draw = ImageDraw.Draw(walls)
-        for x in range(world.size.x):
-            for y in range(world.size.y):
-                tile = world.tiles[x, y]
+        with c.progressbar(world.tiles, length = world.size) as bar:
+            for tile in bar:
                 if tile.wall:
                     if draw_paint and tile.wall.paint:
                         color = tuple(colors["Paints"][str(tile.wall.paint)])
                     else:
                         color = tuple(colors["Walls"][str(tile.wall.type.value)])
                     draw.point((x, y), color)
```

Comment:
Here are the now-undefined variables.

Reply:
Instead of using for-loop in tile for bar, should I restore the nested for loop which defined x and y in the original code?

Reply:
Yes, it is needed by the `draw.point((x, y), color)` function ^^

Reply:
Okay thanks :)

Reply:
I have made the chances.